lib/components/chat_history/twitch/subscription_event.dart

```diff
@@ -72,6 +72,16 @@ class TwitchSubscriptionMessageEventWidget extends StatelessWidget {
   const TwitchSubscriptionMessageEventWidget(this.model, {Key? key})
       : super(key: key);
 
+  Color chipBackgroundColor(String tier, BuildContext context) {
+    if (tier == '2000') {
+      return  const Color(0xFFFFEA00);
```

Comment:
```suggestion
      return const Color(0xFFD2D2E6);
```
Tier 2 is not commonly used lets change it to the lighter color.